docs/quickstart/installing-thundernetes.md

```diff
@@ -0,0 +1,83 @@
+---
+
+# Installing Thundernetes
+
+Follow the steps below to install Thundernetes on your Kubernetes cluster.
+
+## Install cert-manager
+
+Once you have a Kubernetes cluster up and running, you need to install [cert-manager](https://cert-manager.io). Cert-manager is a certificate controller for Kubernetes and it is needed for the webhooks used to validate your GameServerBuilds.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
+```
+
+To verify that cert-manager is installed, you can run the following command:
+
+```bash
+kubectl get pods -n cert-manager
+```
+
+## Install Thundernetes with the installation script
+
+You can run the following command to install Thundernetes. 
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/PlayFab/thundernetes/main/installfiles/operator.yaml
+```
+
+**Note:** installing Thundernetes will automatically deploy two DaemonSets: one for Linux nodes and for Windows nodes. If you only plan to use one OS for the nodes, you can safely delete the DaemonSet for the other. These DaemonSets live under the ```thundernetes-system``` namespace, you can optionally delete them with the following commands (even though there is no harm in keeping them around):
+
+- Windows: ```kubectl delete -n thundernetes-system daemonset thundernetes-nodeagent-win``` (if you plan to only use Linux game servers)
+- Linux: ```kubectl delete -n thundernetes-system daemonset thundernetes-nodeagent``` (if you plan to only use Windows game servers)
+
+To verify that Thundernetes is up and running, you can run the following command:
+
+```bash
+kubectl get pods -n thundernetes-system
+```
+
+You should see something like that, for a 3-node cluster:
+
+```bash
+NAME                                               READY   STATUS    RESTARTS   AGE
+thundernetes-controller-manager-5fc55b9db7-rcns9   1/1     Running   0          10s
+thundernetes-nodeagent-6wljt                       1/1     Running   0          15s
+thundernetes-nodeagent-6x8c4                       1/1     Running   0          20s
+thundernetes-nodeagent-eabgh                       1/1     Running   0          17s
+```
+
+At this point, you are ready to run a test game server on Thundernetes to verify that the system is working as expected. If you want to run one of our sample game servers, check our [samples](samples.md). Otherwise, if you want to run your own game server, go to [this document](../developertool.md).
+
+The aforementioned scripts install Thundernetes with unauthenticated access to the allocation API service. This is fine for development scenarios, but for production environments you would need to secure the service. There are a couple of options you can use. Thundernetes offers a way to configure mTLS authentication to the allocation API service, you can read the next section. Alternatively, you can use a reverse proxy, like [nginx-ingress](https://github.com/kubernetes/ingress-nginx).
```

Comment:
"Alternatively, you can use a reverse proxy" an Ingress can be a reverse proxy, but it also does other stuff, in the mTLS case you aren't even using the reverse proxy. Maybe just say "Alternatively, you can use a Kubernetes Ingress".